routes/api.js

```diff
@@ -93,10 +100,14 @@ router.post('/login', [
   if (!ret.success) return res.status(500).json(ret);
   else if (ret.data.length == 0) return res.json({success: false, message: "The email is incorrect."});
   // check password
-  const hashedPwd = ret.data[0].password;
+  const user = ret.data[0];
+  const hashedPwd = user.password;
   if (!bcrypt.compareSync(password, hashedPwd))
     return res.json({success: false, message: "The password is incorrect."});
-  return res.json(ret);
+  // issue token
+  const payload = {usersId: user.usersId, email: user.email};
+  const token = jwt.sign(payload, jwtSecret, {expiresIn: '2h'});
```

Comment:
Old tokens can be used if an organizer logs in multiple times. Should there only be a 'lastest' token that an organizer can use?

Reply:
fix this after the database is chosen loooooool